niceman/retrace/tests/test_packagemanagers.py

```diff
@@ -80,7 +80,7 @@ def test_dpkg_manager_identify_packages():
             assert o.label, "A non-local origin needs a label"
             assert o.site, "A non-local origin needs a site"
             assert o.archive_uri, "An archive_uri should have been found"
-            assert o.date, "An package should have been found"
```

Comment:
irrelevant to this PR but "duplication is evil".  use `.__attrs_attrs__` and `getattr` to get them in a loop ;)